Frameworks/include/MockClass.h

```diff
@@ -503,11 +527,20 @@ VERIFY_ARE_EQUAL(1, callCount);
 #if !defined CUSTOM_MOCK_CALLBACK_VALIDATOR
 namespace Test {
 namespace Mock {
+#ifdef VERIFY_IS_NOT_NULL
 template <typename TFunctor>
```

Comment:
This seems pretty hacky. What happens if this is included before the test framework headers? Is there anything better we can do here?